cmd/knit/cmd/irqaff.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-kni/debug-tools/pkg/irqs"
+	softirqs "github.com/openshift-kni/debug-tools/pkg/irqs/soft"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
+)
+
+type irqAffOptions struct {
+	checkEffective bool
+	checkSoftirqs  bool
+}
+
+func newIRQAffinityCommand() *cobra.Command {
+	opts := &irqAffOptions{}
+	irqAff := &cobra.Command{
+		Use:   "irqaff",
+		Short: "show IRQ/softirq thread affinities",
```

Comment:
Would be nice a few examples with the expected output when seeing the usage

Reply:
Will add to the README.

Reply:
ok